tests/e2e/kyma_deletion_test.go

```diff
@@ -3,6 +3,8 @@ package e2e_test
 import (
 	"os/exec"
 
+	apimetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Please rename this test to indicate it's testing delete kyma by foreground.

Reply:
no, this test is not designed to test foreground, it's testing kyma get deleted and module under stuck, then Kyma CR only get deleted when Kubeconfig Secret get deleted. This one should also test against background delete.